doc/src/main/docs/elasticsearch.asciidoc

```diff
@@ -684,6 +684,87 @@ TIP: You can use the `POST {apiLatest}/utilities/validateSchema` endpoint to val
 
 TIP: If you just want to add your field to the source document you can create a mapping with `"index": false`.
 
+[[language-override]]
+=== Mappings / Settings per Language
+
+It is also possible to define different settings / mappings for different languages. To do so, add the `_meshLanguageOverride` property to the `elasticsearch` object.
+In this object you can override settings / mappings for each language. You can also provide the same setting / mapping for multiple language by using a
+comma separated list of languages as the key.
+
+For each overridden language, Gentics Mesh will create additional indices in Elasticsearch. Node contents of that language will be saved in that index.
+When a node content is created with a language that is not overridden, the node content will be saved in the default index. The settings / mappings of the
```

Comment:
```suggestion
comma separated list of languages as the key.

NOTE: For each overridden language, Gentics Mesh will create additional indices in Elasticsearch. This might affect the resources used by Elasticsearch.

 Node contents of that language will be saved in that index.
When a node content is created with a language that is not overridden, the node content will be saved in the default index. The settings / mappings of the
```